src/Favicon/Favicon.php

```diff
@@ -311,9 +314,13 @@ private function checkImageMType($url) {
     private function checkImageMTypeContent($content) {
         if(empty($content)) return false;
 
-        $fInfo = finfo_open(FILEINFO_MIME_TYPE);
-        $isImage = strpos(finfo_buffer($fInfo, $content), 'image') !== false;
-        finfo_close($fInfo);
+        $isImage = true;
+        try {
+            $fInfo = finfo_open(FILEINFO_MIME_TYPE);
+            $isImage = strpos(finfo_buffer($fInfo, $content), 'image') !== false;
+            finfo_close($fInfo);
+        } catch (Exception $e) {
```

Comment:
Fixed in https://github.com/ArthurHoaro/favicon/pull/5/commits/b4308ffe168f19c54b368808023f5b0bfa1b62aa